hw/pci-bridge/simbridge.c

```diff
@@ -295,12 +299,12 @@ simdevice_memrd(void *opaque, hwaddr addr, unsigned size)
     /* add bar as model expects physical address */
     addr += simbar->sd->bar[baridx].addr;
     if (simc_memrd(bdf, baridx, addr, size, &val) < 0) {
-        dbgprintf("simdevice_memrd(0x%"PRIx64", 0x%x) failed\n",
-                  addr, size);
+        dbgprintf("simdevice_memrd(0x%"PRIx64", 0x%x) failed %s\n",
+                  addr, size, WARN_UNALIGNED(addr));
```

Comment:
In the normal aligned case this will print a trailing space.  Can you add the space to " (unaligned)" message for a little cleaner output in the normal case?

Reply:
Yeah, good catch. I will update this on the next revision. Thanks.